internal/driver/webui.go

```diff
@@ -210,6 +210,9 @@ func defaultWebServer(args *plugin.HTTPServerArgs) error {
 	mux := http.NewServeMux()
 	mux.Handle("/ui/", http.StripPrefix("/ui", handler))
 	mux.Handle("/", redirectWithQuery("/ui", http.StatusTemporaryRedirect))
+	// Enable self-profiling handlers if net/http/pprof is imported by the main
+	// program.
+	mux.HandleFunc("/debug/pprof/", http.DefaultServeMux.ServeHTTP)
```

Comment:
Should we apply the same auth checks (localhost etc.) that we apply to other handlers?